style.css

```diff
@@ -835,3 +849,251 @@ p {
   text-align: center;
   color: #fff;
 }
+
+.end {
+  position: relative;
+  width: 74px;
+  height: 74.02px;
+  right: -19px;
+  top: 3793px;
+}
+
+.next-step {
```

Comment:
- [ ] Kindly make sure your page fits the screen size on the mobile version. We do not need to scroll to see the entire page width. consider fixing that.
Your design :point_down: 
![Screenshot from 2022-11-01 19-07-57](https://user-images.githubusercontent.com/97100091/199535596-39c7d184-cfdb-4fc0-b7c7-2eb40bf5aab2.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199535737-4fe94c59-329b-4246-b479-4a81f0603448.png)

Reply:
Hi @christianbiring1, can you please review this comment again? Because the above screenshot provided isn't my project.